uportal-war/src/main/resources/logback.xml

```diff
@@ -146,11 +146,11 @@
   </logger>
   -->
   
-  <!-- Uncomment to see the XML at various stages in the rendering pipeline
+  <!-- Uncomment to see the XML at various stages in the rendering pipeline -->
```

Comment:
@apetro As always, **great** attention to detail!  I appreciate that about your reviews.  Glad you caught this.